README.md

```diff
@@ -83,6 +85,15 @@ below to listen to these events.
 This method closes the picker without the user's action.
 Make sure you are not ruining user experience unnecessarily.
 
+#### `simplepicker.reset(date)`:
+
```

Comment:
For consistency with the format of other methods let's document the method parameters:
```suggestion
  * `date` (required, `Date`) - the date to set after the reset.
```